dockstore-webservice/src/main/resources/openapi3/openapi.yaml

```diff
@@ -7651,8 +7652,7 @@ components:
         name:
           type: string
         onlineProfileId:
-          type: integer
-          format: int64
+          type: string
```

Comment:
question: there's a jsonignore on this above, why does this even show up here?

Reply:
This is on the user_profile, but it's only jsonignored on the token. I guess it should be ignored on both.